examples/nextjs/.gitignore

```diff
@@ -11,20 +11,24 @@
 # next.js
 /.next/
 /out/
-!public/
 
 # production
 /build
 
 # misc
 .DS_Store
-.env*
+*.pem
 
 # debug
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-# Environment Variables
-.env
-.env.build
+# local env files
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```

Comment:
This could probably use a wildcard to ignore all local files:

```
env.local
.env.*.local
```

Reply:
@styfle True and it's intentional. We made this explicit for now in create-next-app